pytest.ini

```diff
@@ -0,0 +1,9 @@
+[pytest]
+# customize test discovery to treat benchmarks as tests
+python_files = *.py
+python_classes = Time *Benchmarks
+python_functions = time_*
```

Comment:
For future-proofness, we might want to support these different asv benchmark types:

https://asv.readthedocs.io/en/stable/benchmarks.html

specifically ``time_*``, ``timeraw_*``, ``mem_*``, ``peakmem_*``, and ``track_*`` although the latter might not work because I think pytest doesn't like tests that return things.